cmd/state-installer/cmd.go

```diff
@@ -132,11 +132,6 @@ func main() {
 				Description: "Defaults to 'release'.  Specify an alternative channel to install from (eg. beta)",
 				Value:       &params.branch,
 			},
-			{
-				Shorthand: "b", // backwards compatibility
-				Hidden:    true,
-				Value:     &params.branch,
-			},
```

Comment:
Good catch! There's also the channel flag though. And the branch definition itself in the types.